### ISSUE TITLE
Display user advice when creating new component entry

### ISSUE THREAD

**Issue by mcjaeger** [sw360/sw360portal#456]
People just create their component, without caring for existing records.

There should be a (Dialogue box) warning which advises the user before creating a new component to really use the search for existing components.

Maybe displaying the warning or not could be a setting in sw360properties.

2. Clarification from Discussion:

Actually having the warning message at "Add Component" could lead to auto-click-away, rather have the proposals when the user enters a new name for the component.

Proposal is to have search once the focus moves away from the name text field and display search results to the user.

If the system found entries, then a dialogue box displays the found entries (best 10).

The user can then either a) ignore or b) jump to the matching component.

**Comment by imaykay** [comment]
On the current feature branch (see #528) the popup occurs after bluring the component name field and looks like this (having entered "logging" in the field):

<img width="1600" alt="grafik" src="https://user-images.githubusercontent.com/27430165/29062401-dc3a5e68-7c21-11e7-9876-ac507b287299.png">

It also comes up when editing the component name in an existing component! 

And in addition it even pops when no similar components could be found. The reason for this is as users might getting used to see alternatives, they might be confused when nothing happens after leaving the name field. But that is just my point of view and might be subject to change.

**Comment by alexbrdn** [comment]
I don't pretend to be an expert in English language, but I think there's something amiss with the dialog title. How about this wording: "To avoid duplicate components, check these similar ones! Does yours already exist?"

**Comment by imaykay** [comment]
@alexbrdn Me neither, so I just go with your suggestion :-)
